jenkins/bootstrap_test.py

```diff
@@ -1461,9 +1461,8 @@ def Check(job, name):
             self.assertIn('frequency', job)
             self.assertIn('trigger-job', job)
             self.assertNotIn('branch', job)
-            self.assertIn('json', job)
             modern = is_modern(name)
-            self.assertEquals(modern, job['json'])
+            self.assertEquals(modern, job.get('json', 1), name)
```

Comment:
This is a deceptive unit test, as setting json: 0 in the yaml does not change the fact that the bootstrap.py flag remains `--json` and is therefore always true